templates/articles/_add_comment.html

```diff
@@ -0,0 +1,27 @@
+<turbo-frame id="add_comment">
+{% if user.is_authenticated %}
+    <form action="{% url 'add_comment' article.slug %}" class="card comment-form" method="post">
+    {% csrf_token %}
+        <div class="card-block">
+                <textarea
+                        class="form-control"
+                        placeholder="Write a comment..."
+                        rows="3"
+                        name="body"
+                ></textarea>
+        </div>
+        <div class="card-footer">
+            <img
+                    src="{{ user.profile.img }}"
+                    class="comment-author-img"
+            />
+            <span class="comment-author">Commenting as <a href="">{{ user }}</a></span>
```

Comment:
There's a missing link here:
```suggestion
            <span class="comment-author">Commenting as <a href="{% url 'profile_view' request.user.username %}">{{ user }}</a></span>
```